roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -246,6 +261,30 @@
     - "{{ tempdir }}/templates/kibana-dc.yaml"
     delete_after: true
 
+- name: Generate Memcached DC template
+  template:
+    src: memcached.j2
+    dest: "{{ tempdir }}/templates/memcached-dc.yaml"
+  vars:
+    component: "{{ kibana_component }}"
+    logging_component: kibana
+    deploy_name: "memcached-{{ kibana_name }}"
+    image: "{{ openshift_logging_image_prefix }}logging-memcached:{{ openshift_logging_image_version }}"
+    kibana_cpu_limit: "{{ openshift_logging_kibana_cpu_limit }}"
```

Comment:
The key here is we need to have the `requests` and `limits` for the memory size configuration of memcached be reflected properly.